docs/site/Relations.md

```diff
@@ -0,0 +1,164 @@
+---
+lang: en
+title: 'Relations'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Relations.html
+summary:
+---
+
+## Overview
+
+Model relation in LoopBack 3 is one of its powerful features which help users
+define real-world mappings between their models, access sensible CRUD APIs for
+each of the models, and add querying and filtering capabilities for the relation
+APIs after scaffolding their LoopBack applications. In LoopBack 4, with the
+introduction of [repositories](Repositories.md), we aim to simplify the approach
+to relations by creating constrained repositories. This means that, based on the
+relation definition, certain constraints need to be honoured by the target model
+repository, and thus, we produce a constrained version of it as a navigational
+property on the source repository. The first relation type we have available is
+the `has many` relation which operates by constraining the target repository by
```

Comment:
We should probably just call `has many` `HasMany` like we do in lb3 documentation.

Reply:
I also think that we should probably extract out `HasMany` into a 'relations list' where we show what relations lb4 supports. Something like this:
```md
Supported relations:
- [HasMany](HasMany-relations.md or something along that line)
```